.github/workflows/test_deploy.yml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - master
+      - release/*
```

Comment:
add build trigger when pushing to release branches to make it easier for testing